.github/workflows/build-publish-main.yml

```diff
@@ -0,0 +1,48 @@
+name: Mock Bot Build and Publish Pipeline
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  PackageDeploy:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Docker Setup BuildX
+      uses: docker/setup-buildx-action@v2
+
+    - name: Set image tag
+      run: |
+        LOWER_CASE_GITHUB_REPOSITORY=$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]')
+        echo "DOCKER_TAG_SHA=ghcr.io/${LOWER_CASE_GITHUB_REPOSITORY}:${GITHUB_SHA}" >> $GITHUB_ENV
+    
+    - name: Docker Build
+      working-directory: src/MockClassifier.Api/
```

Comment:
Should this be the path to the mock bot?

Reply:
My bad, missed one in copy-paste action.